src/test/run-pass/mir_trans_calls.rs

```diff
@@ -88,6 +88,19 @@ fn test8() -> isize {
     Two::two()
 }
 
+#[link(name = "c")]
+extern {
+    fn printf(format: *const u8, ...) -> i32;
```

Comment:
Probably the most obvious VA_ARG function to test is `syscall`, but it is system-dependent (duh).